### PR TITLE
Adicionando arquivo .env

### PR DESCRIPTION
**Pull request by BrunoAAM**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+DB_UserName=root
+DB_Password=password
+DB_Database=digitalStore
+DB_Host=127.0.0.1
+DB_Dialect=mysql
```
Comment (carlosxjm) [inline .env.example]: Add blank line at the end of the file.

Comment (carlosxjm) [reply .env.example]: Please, check all files with the same issue.

**Review by carlosxjm — CHANGES_REQUESTED** [review]
Would be great to add some instructions to the README file .
Just a suggestion how the user should create the env file copy:
```
cp .env.example .env
```

**Review by Gabriel-Vasconcelos — APPROVED** [review]
brabo

**Review by carlosxjm — APPROVED** [review]

**Pull request merged**